MqttClient_Hello/app/application.cpp

```diff
@@ -11,12 +11,25 @@
 void startMqttClient();
 void onMessageReceived(String topic, String message);
 
-Timer procTimer;
+Timer procTimer, mqttTimer;
 
 // MQTT client
 // For quickly check you can use: http://www.hivemq.com/demos/websocket-client/ (Connection= test.mosquitto.org:8080)
 MqttClient mqtt("test.mosquitto.org", 1883, onMessageReceived);
 
+// Check for MQTT Disconnection
+void IRAM_ATTR checkMQTTDisconnect(TcpClient& client, bool flag){
```

Comment:
Is there a good reason for putting this in ram? ram is very limited on esp8266, and it can;t see why this would need to be there.